types/testing-library__vue/testing-library__vue-tests.ts

```diff
@@ -0,0 +1,146 @@
+component.findAllByText("foo"); // $ExpectType Promise<HTMLElement[]>
+component.findAllByText(/some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType Promise<HTMLElement[]>
+
+lib.cleanup(); // $ExpectType void
+
+// Reexports queries from dom-testing-library
+lib.getByLabelText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType HTMLElement
+lib.getByPlaceholderText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByAltText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByTitle(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByDisplayValue(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByRole(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByTestId(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x}); // $ExpectType HTMLElement
+lib.getByText(elem, "foo"); // $ExpectType HTMLElement
+lib.getByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType HTMLElement
+lib.getAllByText(elem, "foo"); // $ExpectType HTMLElement[]
+lib.getAllByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType HTMLElement[]
+lib.queryByText(elem, "foo"); // $ExpectType HTMLElement | null
+lib.queryByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType HTMLElement | null
+lib.queryAllByText(elem, "foo"); // $ExpectType HTMLElement[]
+lib.queryAllByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType HTMLElement[]
+lib.findByText(elem, "foo"); // $ExpectType Promise<HTMLElement>
+lib.findByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType Promise<HTMLElement>
+lib.findAllByText(elem, "foo"); // $ExpectType Promise<HTMLElement[]>
+lib.findAllByText(elem, /some text/, {exact: true, trim: false, collapseWhitespace: true, normalizer: x => x, selector: '*'}); // $ExpectType Promise<HTMLElement[]>
+
+// Reexports event functions from dom-testing-library
+// Note that it does NOT make the core fireEvent() function asynchronous
+lib.fireEvent(elem, new Event('change')); // $ExpectType boolean
```

Comment:
AFAIK, `fireEvent()` is returning a Promise ([source](https://github.com/testing-library/vue-testing-library/blob/master/src/vue-testing-library.js#L102)), and here it appears to be returning a boolean 

disclaimer: I might be missing something basic, as I have no experience with type definitions 😇

Reply:
Yeah, I'll get this updated to 2.0 after we get 1.2 published. (Or I could just do that now if we want to avoid another PR—I don't feel that strongly about it.)

Reply:
2.0 is already published and available, so if you feel like changing it now, I don't see why not. 👍